### ISSUE TITLE
Scheduler week view multi-day events

### ISSUE THREAD

**Issue by danielkaradachki** [telerik/kendo-ux#10]
Currently the multi-day events are rendered in the all day slots. Seems to be a common requirement to render them in the times content which arises two questions:

1. How to indicate that the rendered element continues in the next/previous slot.
2. How to layout the events so that it is obvious that they are the same.

Based on the current design and layout the result looks a bit strange:
![multiday](https://user-images.githubusercontent.com/1334167/51021061-1ba4d180-1589-11e9-858d-6f48d30077a5.png)

Will appreciate any suggestions on how to improve it.
cc @tsvetomir @lkarakoleva

**Comment by lkarakoleva** [comment]
The up/down arrow indicator is better to be center aligned, as @tsvetomir mentioned, it is not necessary to have resize handler there. 

[ASP.NET AJAX scheduler](https://demos.telerik.com/aspnet-ajax/scheduler/examples/day-week-multi-day-views/defaultcs.aspx) has multi-days events with up/down arrow indication. 
- If the event is all day long event it should be visualized in the all-day slot. 
- If the event is not an all day event, it is not presented in the time slots
- If the event starts in one day, continuous all day during next day and then ends in third day we have both visualization in all-day slot and in the timeline slot. We have left/right arrow indication in the all-day event if it is a multi-day:
For instance, 'DevMedia conference' in the image below:
<img width="1491" alt="screen shot 2019-01-11 at 11 02 47 am" src="https://user-images.githubusercontent.com/16577622/51023874-b81ea200-1590-11e9-8d88-6ff5da7caa8a.png">

 Both Outlook and Google Calendar visualized multi-days events in all-day slots, even though they are not all-days and labeled the start hour:
<img width="735" alt="screen shot 2019-01-11 at 11 14 54 am" src="https://user-images.githubusercontent.com/16577622/51024585-a0481d80-1592-11e9-98df-c8524020d989.png">
<img width="507" alt="screen shot 2019-01-11 at 11 15 12 am" src="https://user-images.githubusercontent.com/16577622/51024592-a5a56800-1592-11e9-821b-082dddd7a380.png">

**Comment by tsvetomir** [comment]
Whether or not the event goes to the all-day slot will be up to the developer. If the `isAllDay` flag is `true` it will be rendered there regardless of it's start and end dates.

The mixed case where only part of middle section goes in the all-day leads to excessive fragmentation. Maybe we can indicate the beginning of the event by scaling it to occupy only a portion of the all-day slot in the first and last days:

| 01.01 | 02.01 | 03.01 |
| ---: | :---: | :--- |
|  === | ======  |  = |

**Comment by lkarakoleva** [comment]
I agree, if we show all multi-day events in all-day slot, it is an intuitive indication to scale the event according to the portion of the day it takes.

**Comment by tsvetomir** [comment]
Revisiting this issue, I think that scaling events in the all-day slots has a fundamental problem. It reduces the real estate of the event element, potentially down to a few pixels (isAllDay = true, 23:55 - 00:05).

Maybe the approach with printing out the start time is not so bad and can be easily customized through templates.
